src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
+const DEFAULT_ANIMAL_HEALTH = 100;
+const CARNIVORE_DAMAGE = 50;
+
 class Animal {
-  // write your code here
+  constructor(name, health = DEFAULT_ANIMAL_HEALTH) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, isHidden = false) {
```

Comment:
if a function begins to take more than two arguments, then it is better to combine these arguments into a single object being passed
this is more convenient for developers and allows the class to be extended in the future